blosc/schunk.c

```diff
@@ -95,6 +95,11 @@ blosc2_schunk *blosc2_new_schunk(blosc2_cparams cparams, blosc2_dparams dparams,
   dparams.schunk = schunk;
   schunk->dctx = blosc2_create_dctx(dparams);
 
+  // Storage info.  We use defaults here because we don't want to break
+  // the existing `blosc2_new_schunk()` API.
+  schunk->storage = (blosc2_storage*)malloc(sizeof(blosc2_storage));
+  memcpy(schunk->storage, &BLOSC2_STORAGE_DEFAULTS, sizeof(blosc2_storage));
+
   schunk->frame = frame;
   if (frame != NULL) {
```

Comment:
You should update `schunk->storage` params to indicate that a frame (persistent or not) is being used.

Reply:
Addressed in 8db4718